### PR TITLE
Increase coverage

### PR DESCRIPTION
**Pull request by Timthetic**
Adds tests for edge cases and missing use case

**Makes a few changes to meetup.go**

* You should look over meetup.go
* You can **skip** meetup_test.go if you want.

### CODE REVIEW

File: server/impl/meetup.go [hunk 1]
```diff
@@ -298,10 +299,7 @@ func (i *Implementation) PostMeetup(params operations.PostMeetupParams, _ interf
 	}
 
 	modelMeetup := modelMeetupRequestBodyToModelMeetup(params.Meetup, id.(string))
-	if err := i.modelMeetupToDBMeetup(&dbMeetup, &modelMeetup); err != nil {
-		logger.WithError(err).Error("Failed to create db meetup object")
-		return responders.InternalServerError{}
-	}
+	_ = i.modelMeetupToDBMeetup(&dbMeetup, &modelMeetup)
```
Comment (Timthetic) [inline server/impl/meetup.go:302]: modelMeetupToDBMeetup never returns an error

File: server/impl/meetup.go [hunk 2]
```diff
@@ -595,7 +590,7 @@ func (i *Implementation) PatchMeetupIDAttendee(params operations.PatchMeetupIDAt
 	id := session.Values[UserID]
 	attendeeID, err := db.UserIDFromString(params.PatchMeetupAttendeeBody.Attendee)
 	status := params.PatchMeetupAttendeeBody.AttendeeStatus
-	if attendeeID != 0 && err != nil {
+	if err != nil && params.PatchMeetupAttendeeBody.Attendee != "" {
```
Comment (Timthetic) [inline server/impl/meetup.go:593]: If someone put in "as;dlfkj" as an ID, this used to not trigger because the error condition [ParseUint](https://golang.org/pkg/strconv/#ParseInt)

File: server/impl/meetup.go (hunk 1 above)
Comment (conniec218) [reply server/impl/meetup.go:302]: nice catch, should we also update the function signature then? I think it still says it returns an error

File: server/impl/impl_test.go [hunk 3]
```diff
@@ -142,12 +143,27 @@ func populateDatabase(i *Implementation) {
...
+	TestMeetupRemote = &db.Meetup{
+		Title:       "Cry",
+		Time:        time.Now().Add(24 * time.Hour),
+		Description: "",
+		Tags:        nil,
+		MaxCapacity: 10,
+		MinCapacity: 1,
+		Owner:       TestUserFriend.ID,
+		Location: db.MeetupLocation{
+			URL: "https://hack.uclaacm.com/",
```
Comment (jamieliu386) [inline server/impl/impl_test.go:155]: We love crying at ACM Hack @ UCLA :)

File: server/impl/meetup.go (hunk 2 above)
Comment (TimothyGu) [inline server/impl/meetup.go:593]:
```suggestion
	if err != nil || params.PatchMeetupAttendeeBody.Attendee != "" {
```

Comment (TimothyGu) [reply server/impl/meetup.go:593]: Ignore

**Review by TimothyGu — APPROVED** [review]
